src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -19,6 +19,13 @@
 
 REPO_CENTRAL = b'https://hg.mozilla.org/mozilla-central'
 REPO_REVIEW = b'https://reviewboard-hg.mozilla.org/gecko'
+MAX_COMMENTS = 30
+MOZREVIEW_COMMENT = '''
+Static analysis found {} code defects in this patch{}.
+
+- You can run this analysis locally with: ./mach static-analysis check path/to/file.cpp
+- If you notice a problem in this automated review, please report it here : http://bit.ly/2y9N9Vx
```

Comment:
Nit: No need for the `- ` actually. Also, please use `` backticks for the `./mach` command. And please no space before the `:`.

E.g.:

```
Static analysis found {} C/C++ defects in this patch{}.

You can run this analysis locally with: `./mach static-analysis check path/to/file.cpp`

If you see a problem in this automated review, please report it here: http://bit.ly/2y9N9Vx
```

Reply:
Ultra-nit: It would be nice to only add the `s` to `defects` when `nb !== 1`.